src/components/OutfitMakerComponent.tsx

```diff
@@ -1,9 +1,267 @@
+  const [savedOutfits, setSavedOutfits] = useState<SavedOutfit[]>([]); // State to store saved outfits
+  const [showSavedOutfits, setShowSavedOutfits] = useState(false); // State to toggle views
+  const [saveConfirmation, setSaveConfirmation] = useState(false); // Confirmation state
+
+  const [selectedCategory, setSelectedCategory] = useState<string | null>(null);
+  const [selectedSubCategory, setSelectedSubCategory] = useState<string | null>(null);
+
+  const [modalOutfit, setModalOutfit] = useState<SavedOutfit | null>(null); // State for modal outfit
+
+  // Toggles the visibility of the category menu
+  const toggleMenu = () => setShowMenu((prev) => !prev);
+
+  // Sets the category and shows sub-categories
+  const handleCategorySelect = (category: string) => {
+    setSelectedCategory(category);
+    setSelectedSubCategory(null); // Reset selected subcategory when category changes
+  };
+
+  // Resets to the top-level categories
+  const handleBackToCategories = () => {
+    setSelectedCategory(null);
+    setSelectedSubCategory(null);
+  };
+
+  // Adds a selected item to the canvas with random position
+  const handleItemSelect = (item: { name: string; image: string }) => {
+    setSelectedItems((prev) => [
+      ...prev,
+      {
+        ...item,
+        x: Math.random() * 400, // Random position for x (within canvas width)
+        y: Math.random() * 300, // Random position for y (within canvas height)
+      },
+    ]);
+    setShowMenu(false); // Close menu after item selection (if intended)
+  };
+
+  // Deletes an item from the canvas
+  const handleDeleteItem = (index: number) => {
+    setSelectedItems((prev) => prev.filter((_, i) => i !== index));
+  };
+
+  // Updates item position after dragging
+  const handleUpdateItemPosition = (index: number, x: number, y: number) => {
+    setSelectedItems((prev) =>
+      prev.map((item, i) =>
+        i === index ? { ...item, x, y } : item
+      )
+    );
+  };
+
+  // Handle saving the outfit
+  const handleSaveOutfit = () => {
+    const newOutfit: SavedOutfit = {
+      id: new Date().toISOString(), // Use the current time as a unique id
+      items: selectedItems,
+    };
+    setSavedOutfits((prevOutfits) => [...prevOutfits, newOutfit]);
+    setSelectedItems([]); // Clear selected items after saving
+
+    // Show confirmation message
+    setSaveConfirmation(true);
+    setTimeout(() => setSaveConfirmation(false), 2000); // Hide after 2 seconds
+  };
+
+  // Toggle between create outfit view and saved outfits view
+  const toggleSavedOutfitsView = () => {
+    setShowSavedOutfits((prev) => !prev);
+  };
+
+  return (
+
+//   <Box
+//   alignItems="center"
+//   justifyContent="space-between"
+//   flex={1}
+//   margin={4} // Adjust margin to ensure spacing
+//   w="100%" h="100vh" bg="white" safeArea
+// >
+
+<Box 
+
+w="100%" 
+maxW="100vw" 
+overflow="hidden"
+px={2} // reduced padding for mobile
+>
```

Comment:
```suggestion
maxH="80vh"
>
```

Reply:
I think this might fix the issue you were having with the container getting hidden by the Navbar.